src/components/FavoriteIcon/index.ts

```diff
@@ -0,0 +1,43 @@
+import BaseComponent from "../BaseComponent";
+import { MENU_APP_EVENTS } from "../../constants/event";
+import { toggleFavoriteStateByName } from "../../domains/Restaurants";
+import { filledFavoriteIcon, linedFavoriteIcon } from "../../assets/index";
+
+class FavoriteIcon extends BaseComponent {
+  protected render() {
+    const isFavorite = this.getAttribute("is-favorite") === "true" ? true : false;
```

Comment:
간결하게 줄일 수 있을 것 같아요 :)

```suggestion
    const isFavorite = this.getAttribute("is-favorite") === "true";
```